index.html

```diff
@@ -73,20 +73,14 @@
 		</nav>
 	</header>
 
-
-
-
-
-
-
 	<div class="hero">
 		<div class="container">
 			<span class="hero-background iconNew-fu-logo-icon"></span>
 			<div id="hero-download" class="pull-right">
 				<h1>Write better tests, faster.</h1>
 
-				<a href="https://raw.githubusercontent.com/bitovi/funcunit/v3.1.0/dist/funcunit.js" download="funcunit.js">
-					<button class="download">Download 3.1.0<span>(One file. Everything you need.)</span></button>
+				<a href="https://raw.githubusercontent.com/bitovi/funcunit/v3.2.0/dist/funcunit.js" download="funcunit.js">
+					<button class="download">Download 3.2.0<span>(One file. Everything you need.)</span></button>
```

Comment:
@Rlin415, can you explain this PR a little bit better? This is still hard-coded, so it’s unclear whether the issue is actually fixed or not.